doc/examples/segmentation/plot_chanvese.py

```diff
@@ -0,0 +1,25 @@
+""" This is an example to illustrate the us of the Chan-Vese Algorithm.
+"""
+
+from skimage.data import camera
+from skimage.segmentation.chanvese import chan_vese
+import numpy as np
+import matplotlib.pyplot as plt
+
+def _cv_main():
+    U0 = camera().astype('float')/255.
+    print np.max(U0)
+    # Feel free to play around with these parameters to see how they impact the
+    # result
+    cv = chan_vese(U0, mu=0.5, lambda1=1, lambda2=0.5, tol=1, maxiter=100, dt=100)
+    print ("Chan-Vese algorithm finished after "+str(len(cv[1]))+" iterations.")
+    print cv[1]
+    plt.imshow(cv[0])
+    plt.colorbar()
+    plt.show()
+    plt.plot(cv[1])
+    plt.show()
```

Comment:
This won't work in gallery as expected.
Use `fig, axes = plt.subplots(ncols=2)` as plot on `axes[0]` and `axes[1]`.